lib/learn_test/strategies/jest.rb

```diff
@@ -0,0 +1,91 @@
+module LearnTest
+  module Strategies
+    class Jest < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_jest'
+      end
+
+      def detect
+        package = File.exists?('package.json') ? Oj.load(File.read('package.json'), symbol_keys: true) : nil
+        return false if !package
+
+        if package[:scripts] && package[:scripts][:test]
+          return true if package[:scripts][:test].include?('jest')
+        end
+
+        if package[:devDependencies] && package[:devDependencies][:jest]
+          return true if (package[:devDependencies][:jest].length > 0)
+        end
+
+        if package[:dependencies] && package[:dependencies][:jest]
+          return true if (package[:dependencies][:jest].length > 0)
+        end
```

Comment:
Happy to completely refactor the gem — I was trying to touch as little as possible. Few other things could be DRY-ed up / updated (like the usage of the deprecated `.exists?` instead of `.exist?`).